docs/ops/reference/DUPLICATE_NAMESPACE_SURVEY_2025-10-07.md

```
@@ -0,0 +1,40 @@
+$names | ForEach-Object {
+  Get-ChildItem -Path . -Recurse -Directory |
+    Where-Object { $_.Name -eq $_ } |
+    Select-Object -ExpandProperty FullName
+}
+```
+
+## Summary Table
+
+| Name | Count | Primary Purpose(s) |
+| --- | --- | --- |
+| `assets` (26) | Shared assets under `docs/`, portal page-specific assets, Vite build output (`docs-build`), historical snapshots in `scripts/_*`, and the legacy `src/assets` tree. |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)